server/handlers/load_test_handler.go

```diff
@@ -166,14 +180,75 @@ func (h *Handler) LoadTestHandler(w http.ResponseWriter, req *http.Request, pref
 		return
 	}
 
-	// if values have been passed as body we run test using SMP Handler
-	if string(body) != "" {
+	isSSLCertificateProvided := req.URL.Query().Get("cert") == "true"
+
+	/*
+	 When "cert" query param is set the body contains self-signed certs
+	 and not the SMP config, hence we shouldn't use SMP Handler,
+	 if query param is unset/not present presence of body
+	 if values have been passed as body we run test using SMP Handler
+	*/
```

Comment:
Can we add "cert" as a param in the SMP config?

Reply:
Yes, we should add. // https://github.com/service-mesh-performance/service-mesh-performance/issues/389

Reply:
@MUzairS15 Good!